litestar/plugins/base.py

```diff
@@ -96,6 +101,17 @@ def is_debug_mode(app: Litestar):
         """
 
 
+@runtime_checkable
+class ServerLifespanPluginProtocol(Protocol):
+    """Plugin protocol to extend the CLI Server Lifespan."""
+
+    __slots__ = ()
+
+    @contextmanager
+    def server_lifespan(self, app: Litestar) -> Generator[None, None, None]:
+        yield
+
+
```

Comment:
```suggestion
class CLIPlugin(CLIPluginProtocol):
    """Plugin protocol to extend the CLI Server Lifespan."""

   __slots__ = ()
    
   def on_cli_init(self, cli: Group) -> None:
      """Called when the CLI is initialized. 

    @contextmanager
    def server_lifespan(self, app: Litestar) -> Generator[None, None, None]:
        yield
```

This way we don't need to add another protocol and can extend the base class further without breaking backwards compatibility.

Reply:
I think you are missing the trailing `"""` there in the docstring unless it is hiding from me